src/dev-guide/express-tutorial.md

```diff
@@ -0,0 +1,105 @@
+# Express Tutorial
+
+Express can be used to handle requests to the api of our application, it can also be used as middleware which could help us to pass json to the front end of the application.
+
+
+
+Each express file requires the following head segment:
+
+```
```

Comment:
Add `typescript` or `js` at the end of the opening code block for syntax highlighting

https://rust-lang.github.io/mdBook/format/theme/syntax-highlighting.html